APE24.rst

```diff
@@ -0,0 +1,443 @@
+`listing of accepted packages <https://www.pyopensci.org/python-packages.html#our-accepted-python-open-source-packages>`_.
+As of July 2023, they are in the process of revamping the submission form to handle
+collaborations with different communities (e.g., Astropy), including but not limited to:
+
+* creating a community-specific label for filtering and link to current reviews
+  for that community; and
+* customizing the editorial bot to add community-specific tags.
+
+Is open review a deal breaker?
+------------------------------
+
+We do not think so! In the pre-APE 24 process, the reviewer was anonymous by default
+because we modeled the process after traditional astronomical journals.
+Even then, the reviewer had the option to reveal themselves if they wanted to.
+Given that the Astropy ecosystem is open-source anyway, it makes sense for us
+to move away from the journal-style tradition and embrace a more open process that
+pyOpenSci uses.
+
+Rejecting packages
+------------------
+
+If a package is "rejected," it is likely not in technical or domain scope.
+The rejection will come from pyOpenSci Editor-in-Chief or the editor;
+it does *not* have to come from the Astropy community directly and will be objective.
+Rejection can also happen if the maintainer of the package under review
+is unwilling to ensure the package can be used and maintained.
+While in practice, rejection is rare, usually the Editor-in-Chief would be
+the one doing the rejection.
+
+A package may still be listed under pyOpenSci (and JOSS) if it
+passes the general criteria but fails the Astropy-specific ones;
+e.g., a very nice Python package that has little to do with the Astropy ecosystem.
+
+"Re-review" of packages
+-----------------------
+
+**Note: This does not exist yet and probably will not happen before
+this APE is accepted. As of July 2023, pyOpenSci had a somewhat
+manual process to check in on an accepted package.**
+
+One core goal of pyOpenSci is to support scientific Python packages that are
+maintained over time. Due to the lack of resources, it does not do a *full*
+re-review of packages (involving Editor-in-Chief, editor, and 2 reviewers).
+However, it has plans to set up automated checks that track the "health"
+and maintenance level of a package over time, with the goal of identifying
+packages that have become "orphans" (i.e, unmaintained). Once a package is
+identified as needing additional maintenance, it will be flagged for the community.
+
+"Health" might include frequency of commits, releases, CI status, etc.
+Some, if not most, of this data will be collected using
+`devstats <https://github.com/scientific-python/devstats>`_ and
+`repo-review <https://github.com/scientific-python/repo-review>`_ from
+`Scientific Python`_. This information will be displayed on public dashboards.
+
+However, automation can only go so far; for instance, if automation
```

Comment:
Just a note - i suspect we can figure this out. i believe repo_review can handle gitlab. BUT i like this text as a caveat!